src/js/components/inspectors/Property.tsx

```diff
@@ -76,7 +76,7 @@ function mapStateToProps(reduxState: State, ownProps: OwnProps): StateProps {
   return {
     group:  getIn(state, path + '.group'),
     signal: getIn(state, path + '.signal'),
-    value:  value ? value : ownProps.value,
+    value:  value != null ? value : ownProps.value,
```

Comment:
should this be `!==`?

Reply:
are we guaranteed it will never be undefined? Decided to make it != to deal with undefined but I can change it to !== if that isn't an issue.

Reply:
actually nvm will make the change

Reply:
what about `value && value !== null` ? does that do the same thing? just thinking about how to make the intent clear

Reply:
Sorry I am late to this. So the issue is that value can be 0 so that wouldn't do the same thing. Maybe we don't actually have to worry about undefined so I will change this to !== then.